tuning/trainercontroller/callback.py

```diff
@@ -417,6 +417,7 @@ def on_init_end(
                             config_log_level_str = DEFAULT_TRIGGER_LOG_LEVEL
                     else:
                         control.config = DEFAULT_CONFIG
+                    logger.warning(f"LOG LEVELS: {log_levels}")
```

Comment:
Why do we have a warning level comment here?

Reply:
Addressed.